mentoring/public/js/mentoring_assessment_view.js

```diff
@@ -279,12 +279,16 @@ function MentoringAssessmentView(runtime, element, mentoring) {
     }
 
     function handleSubmitResults(response){
+        submitDOM.attr('disabled','disabled');
```

Comment:
You are right, seems I was really tired last night --- and this is unnecessary, in fact whole fix becomes one-liner.